docs/DeveloperGuide.adoc

```diff
@@ -729,12 +729,69 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |see my application's logo |easily identify the application
+
+|`* * *` |user |add a user with a non-alphabet name |add my friend with a non-alphabet name (e.g. Nguyễn)
+
+|`* * *` |user |use command shortcuts |achieve what I want in a shorter time
+
+|`* * *` |user |add and remove tags |
+
+|`* * *` |user |sort my contacts in A-Z order |see my contacts easier
+
+|`* * *` |user |list contacts with specific tags |
+
+|`* * *` |careless user |see a comfirmation prompt after the 'clear' command |know if I really want to clear my contacts
+
+|`* * *` |user |have a 'quickhelp' command available |view a condensed list of essential instructions
+
+|`* * *` |user |be able to store incomplete addressbook entries |add people to the addressbook even if I do not have all their contact details
+
+|`* * *` |user |store multiple email addresses or phone numbers for a single person |access all a contact's details in a single query
+
+|`* * *` |user |have the screen open at an appropriate size |avoid having to adjust screen sizes manually on each load
+=======
+|`* *` |user |put in phone numbers with dashes and spaces on it |copy-paste a number without having to strip the dashes/spaces
+
+|`* *` |user |adjust the text font size |see better if I have impaired sight
+
+|`* *` |user |search names by regex |search in a more powerful way
+
+|`* *` |user |batch undo/redo (e.g. `undo 3` undoes the last 3 commands) |undo many commands at once to get back to a previous state
+
+|`* *` |user |export my data in formats other than XML (say JSON, CSV) |import them to other 3rd-party apps
+
+|`* *` |user |batch update persons with the same tag |change their details easily
+
+|`* *` |forgetful user |be prompted when a new contact has similar details to an existing person |be warned that I might be adding identical persons twice
```

Comment:
The feature and benefit sound the same. Maybe can make them more clear and distinct?